project-base/src/Shopsys/ShopBundle/Resources/styles/front/common/components/box/chooser.less

```diff
@@ -17,55 +17,30 @@
         display: flex;
         align-items: flex-start;
 
-        .is-no-flex & {
-            display: block;
-            .clearfix();
-        }
-
         &__check {
-            flex: 0 @box-chooser-check-width;
-
-            .is-no-flex & {
-                float: left;
-                width: @box-chooser-check-width;
-            }
+            width: @box-chooser-check-width;
         }
 
         &__image {
-            flex: 0 @box-chooser-image-width;
+            width: @box-chooser-image-width;
             text-align: center;
-
-            .is-no-flex & {
-                float: left;
-                width: @box-chooser-image-width;
-            }
         }
 
         &__title {
-            flex: 1;
+            width: @box-chooser-title-width;
```

Comment:
Is this correct? I think `flex: 1;` does not equal with `width: @box-chooser-title-width;`. Please check it.

Reply:
Fixed adding flex: 1 back. thx

Reply:
Flex: 1 sets all flexible items to be same lenght. Is not necessary bcs, we set calculated width.

Reply:
It is okay for now. We will refactor this later.